src/software/ai/hl/stp/tactic/goalie_tactic.cpp

```diff
@@ -131,7 +131,9 @@ std::optional<Point> GoalieTactic::restrainGoalieInRectangle(
 
 void GoalieTactic::calculateNextIntent(IntentCoroutine::push_type &yield)
 {
-    MoveAction move_action = MoveAction();
+    MoveAction move_action =
+        MoveAction(MoveAction::ROBOT_CLOSE_TO_DEST_THRESHOLD,
+                   MoveAction::ROBOT_CLOSE_TO_ORIENTATION_THRESHOLD, true);
```

Comment:
Can we just remove the default constructor for MoveAction while we're here?

Reply:
#767 will take care of that for multiple Actions and tactics.